src/app/components/JumpTo/README.md

```diff
@@ -1,12 +1,12 @@
-## Description - WORK IN PROGRESS - COMPLETE IN SUBTASK
+## Description
 
-The JumpTo component serves as an in-page navigation menu, similar to a table of contents, allowing users to quickly jump to specific headings within an article. This component renders an `H2`, a list of section titles as `anchor` links, and groups them within a `navigation` landmark. When a link is actioned, the page scrolls down to the relevant section, identified by matching anchor `ids`.
+The JumpTo component serves as an in-page navigation menu, similar to a table of contents, allowing users to quickly jump to specific headings within an article. This component renders an `H2` heading, a list of anchor titles as `anchor` links, and groups them within a `navigation` landmark. When a link is actioned, the page scrolls down to the relevant heading, identified by matching anchor `ids`.
```

Comment:
Don't think it renders an `H2`, its a `strong` element.

Reply:
Changed